common/metrics/queries.go

```diff
@@ -2,48 +2,94 @@ package metrics
+	dataflowAggMemoryUtilMaxQuery = "fetch gce_instance | metric 'compute.googleapis.com/guest/memory/bytes_used' | " +
+		"filter (%s) | group_by 1m, [value_bytes_used_max: max(value.bytes_used)]" +
+		" | every 1m | group_by [], [value_bytes_used_max_max: max(value_bytes_used_max)]"
+	dataflowAggBacklogTimeQuery = "fetch dataflow_job | metric 'dataflow.googleapis.com/job/estimated_backlog_processing_time' | " +
+		"filter && (%s) | group_by 1m, [value_estimated_backlog_processing_time_mean: " +
+		"mean(value.estimated_backlog_processing_time)] | every 1m | group_by [], [value_estimated_backlog_processing_time_mean_mean: " +
+		"mean(value_estimated_backlog_processing_time_mean)]"
+	datastreamAggThroughputQuery = "fetch datastream.googleapis.com/Stream | metric 'datastream.googleapis.com/stream/event_count' | " +
+		"filter (%s) | align rate(1m) | every 1m | group_by [], [value_event_count_aggregate: aggregate(value.event_count)]"
+	datastreamAggUnsupportedEventsQuery = "fetch datastream.googleapis.com/Stream | metric 'datastream.googleapis.com/stream/unsupported_event_count' " +
+		"| filter (%s) | align rate(1m) | every 1m | group_by [], " +
+		"[value_unsupported_event_count_aggregate: aggregate(value.unsupported_event_count)] | group_by 1m, " +
+		"[value_unsupported_event_count_aggregate_mean: mean(value_unsupported_event_count_aggregate)] | every 1m | " +
+		"group_by [], [value_unsupported_event_count_aggregate_mean_aggregate: aggregate(value_unsupported_event_count_aggregate_mean)]"
+	datastreamAggTotalLatencyQuery = "fetch datastream.googleapis.com/Stream | metric 'datastream.googleapis.com/stream/total_latencies' | " +
+		"filter (%s) |  align delta(1m) | every 1m | group_by [], " +
+		"[value_total_latencies_percentile: percentile(value.total_latencies, %s)] | align rate(1m) | every 1m " +
+		"| group_by [], [value_total_latencies_percentile_percentile: percentile(value_total_latencies_percentile, %s)]"
+	pubsubAggSubscriptionSentMessageCountQuery = "fetch pubsub_subscription | metric 'pubsub.googleapis.com/subscription/sent_message_count' " +
+		"| filter (%s ) | align rate(1m) | every 1m | group_by [], " +
+		"[value_sent_message_count_aggregate: aggregate(value.sent_message_count)]"
+	pubsubAggOldestUnackedMessageAgeQuery = "fetch pubsub_subscription | metric 'pubsub.googleapis.com/subscription/oldest_unacked_message_age' " +
+		"| filter (%s) | group_by 1m, " +
+		"[value_oldest_unacked_message_age_max: max(value.oldest_unacked_message_age)] | every 1m | group_by [], " +
+		"[value_oldest_unacked_message_age_max_max: max(value_oldest_unacked_message_age_max)]"
+	gcsAggTotalBytesQuery = "fetch gcs_bucket | metric 'storage.googleapis.com/storage/total_bytes' | " +
+		"filter (%s) | group_by 1m, [value_total_bytes_mean: mean(value.total_bytes)] " +
+		"| every 1m | group_by [], [value_total_bytes_mean_aggregate: aggregate(value_total_bytes_mean)]"
```

Comment:
NIt: Add a comment here between the constants to segregate out aggregated queries and single shard queries. I wouldn't mind putting them in their own files either..